### DISCUSSION TITLE
Regulatory compliance

### DISCUSSION THREAD

**Discussion by dika1383** [wazuh/wazuh#14405] [Q&A] [answered]
Hello !
How to create a new regulatory compliance ? Step by step !

Thanks !

Best regards,

**Comment by mdiego92** [comment, marked as answer]
Hello dika1383,

To set up a custom regulatory compliance, you will need to add it in the rule's `<group>` tag you want to target.

For example, adding '**custom_regulatory_compliance_1.0,**':

```
<rule id="5715" level="3">
  <if_sid>5700</if_sid>
  <match>^Accepted|authenticated.$</match>
  <description>sshd: authentication success.</description>
  <group>authentication_success,pci_dss_10.2.5,custom_regulatory_cumpliance_1.0,</group>
</rule>
```

Then you could filter visualizations or events by "**rule.group: custom_regulatory_compliance_1.0**".

Please let us know if you have any remaining questions or doubts.

Thank you very much.

**Comment by dika1383** [comment]
Hello mdiego92 !
Thanks for your answers.

After creation all group of "'custom_regulatory_compliance_x.x" How to make to make a single regulatory compliance as GPRD on my dashboard ?
Also, In my Server where is locate the file of my customize regulatory compliance ? And how to export ?

Thanks !

**Reply by mdiego92** [reply]
Hello dika1383,

Currently, in order to add new regulatory compliance, you will need to write your own script mapping the data in a way that Wazuh can understand, this would affect the whole ruleset and would be overwritten in the process of a version upgrade.
That's why I recommended, for example, using the `GPRD` dashboard and filtering it by your custom `rule.group`.

Please let me know if you have any remaining questions!

Thank you very much.

**Reply by dika1383** [reply]
Hello mdiego92 !

Ok I test this two method !

I like First method I want to search a way to protect my future new regulatory compliance to version upgrade.

If other people has a solution for this I'm open !

I come back if I have other questions !


Thanks you  very much for your help !

Best regards,
